tox.ini

```diff
@@ -443,6 +445,10 @@ commands_pre =
   kafka-python: pip install opentelemetry-semantic-conventions@{env:CORE_REPO}\#egg=opentelemetry-semantic-conventions&subdirectory=opentelemetry-semantic-conventions
   kafka-python: pip install opentelemetry-sdk@{env:CORE_REPO}\#egg=opentelemetry-sdk&subdirectory=opentelemetry-sdk
   kafka-python: pip install -r {toxinidir}/instrumentation/opentelemetry-instrumentation-kafka-python/test-requirements.txt
+  kafka-python-ng: pip install opentelemetry-api@{env:CORE_REPO}\#egg=opentelemetry-api&subdirectory=opentelemetry-api
```

Comment:
Do you the same problem we have with fastapi-slim so that `kafka-python` pre-commands would run for `kafka-python-ng` too? We resolved by using `fastapislim` without the dash to avoid that.

Reply:
Switched to kafka-pythonng and followed fastapislim's pr here https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2756/files